src/krishna/Assignment_15_Array.java

```diff
@@ -0,0 +1,75 @@
+package krishna;
+
+public class Assignment_15_Array {
+	
+	int maxNum_GivenArray(int[] arr){
+		int max = arr[0];
```

Comment:
If you are taking the 0th index as max number then you can start the loop from index 1 as well, same for other methods